<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterMenu</class>
 <widget class="QDialog" name="RegisterMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>780</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Internship Management App-Register</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="column">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>900</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1200</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 28pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Register based on your occupation</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row">
       <property name="spacing">
        <number>60</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>40</number>
       </property>
       <property name="topMargin">
        <number>80</number>
       </property>
       <property name="rightMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnRegisterStudent">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 85, 127);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Register Student</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRegisterInstructor">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 85, 127);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Register Instructor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRegisterCompany">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 85, 127);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Register Company</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btnGoBack">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(125, 188, 0);
color: rgb(0, 0, 0);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icon_back.jpg</normaloff>icon_back.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
